framework/.prettierrc.json

```diff
@@ -1,5 +1,5 @@
 {
 	"singleQuote": true,
-	"trailingComma": "es5",
+	"trailingComma": "all",
```

Comment:
Why don't we get the template as we do in the libraries? They look alike. https://github.com/LiskHQ/lisk-sdk/blob/development/templates/.prettierrc.json.tmpl